### ISSUE TITLE
Completed Part 3 - made Fixes - Request merge of forked repo into Team6's original repo

### ISSUE THREAD

**Issue by ljhan1** [cmsc388T/git-jiggy-with-it#76]
After all of Team6's functions have been fixed and all feature-item branches have been merged into forked repo's branch "FTR-fix": 
1. Create a PR, add Sagar as reviewer, and request merging the branch "FTR-fix" to branch "main" on the forked repo
2. Once the merge is complete, you should be able to see all of the jobs passing on the Actions tab
3. Open a Pull Request to merge the forked repository with Team6's original repository

DEADLINE: (Friday 01/15)
